scipy/ndimage/_filters.py

```diff
@@ -306,6 +307,11 @@ def gaussian_filter(input, sigma, order=0, output=None,
         for all axes. If specified, the size of the kernel along each axis
         will be ``2*radius + 1``, and `truncate` is ignored.
         Default is None.
+    axes : tuple of int or None, optional
+        If None, it is assumed all axes are to be filtered. Otherwise, only
+        the axes specified in the tuple will be considered. When `axes` is
+        specified, any tuples used for `sigma`, `order`, `mode` and/or `radius`
+        must match the length of `axes`.
```

Comment:
Presumably the `i`th element of these arguments corresponds with the `i`th axis in `axes`? I think that is worth mentioning explicitly, and perhaps a test can checks this (if it's not already)?

Reply:
Yes. Initially I was returning `sorted(axes)` from `_check_axes`, but then I would have to also sort any of these other tuples in the same way which seemed more complicated. I can update the description.